internal/cli/root/atlas/builder.go

```diff
@@ -92,19 +92,7 @@ func Builder(profile *string) *cobra.Command {
 				config.SetService(config.CloudService)
 			}
 
-			if cmd.Name() == figautocomplete.CmdUse { // figautocomplete command does not require credentials
-				return nil
-			}
-
-			if cmd.Name() == "quickstart" { // quickstart has its own check
-				return nil
-			}
-
-			if strings.HasPrefix(cmd.CommandPath(), fmt.Sprintf("%s %s", atlas, "config")) { // user wants to set credentials
-				return nil
-			}
-
-			if strings.HasPrefix(cmd.CommandPath(), fmt.Sprintf("%s %s", atlas, "auth")) { // user wants to set credentials
+			if shouldCheckCredentials(cmd) {
```

Comment:
it says should but if true you don't check it...

Reply:
you are right xD, I will update it